src/geode/geosciences_io/mesh/gocad_common.cpp

```diff
@@ -58,12 +58,18 @@ namespace
                         geode::string_to_double( tokens[3] ),
                         tsurf.crs.z_sign
                             * geode::string_to_double( tokens[4] ) } } );
```

Comment:
Does this z_sign is also used in the .vo input?

Reply:
The z_sign is taken from the header, but not used in the header (the .vo input does not support the grid directions for the moment).